firmware/main.c

```diff
@@ -615,7 +615,7 @@ static void cmd_motor_rt(BaseSequentialStream *chp, int argc, char *argv[]){
         palClearPad(pinPorts[pOpt2].gpio, pinPorts[pOpt2].pin);
 	palClearPad(pinPorts[start].gpio, pinPorts[start].pin);
 
-	num_pulses = atoi(pOpt) * gear_reduction; // Gets the right number of pulses to be done according to the gear reduction
+	num_pulses = atoi(pOpt) * gear_reduction * 2; // Gets the right number of pulses to be done according to the gear reduction, multiplied by 2
```

Comment:
There should be some description as to *why* this needs to be times-2. Also, style-nit, when lines get as long as this, the comment should go *before* the code and be made into a proper paragraph-style comment.